Contribution.md

```diff
@@ -0,0 +1,153 @@
+# Contribution guidelines
+The guidelines for contribution to babitz backend project are listed below. Please go through it before any contribution.
+
+## 1. General guidelines
+- The general guidelines for writing code. We will be using a subset of [typescript contribution guidelines](https://github.com/microsoft/TypeScript/wiki/Coding-guidelines). In case any of these conflict with loopback4 guidelines, loopback4's guideline is taken as priority. 
+
+  ### Names
+    1. Use PascalCase for type names.
+    2. Use PascalCase for enum values.
+    3. Use camelCase for function names.
+    4. Use camelCase for property names and local variables.
+    5. Do not use `_` as a prefix for private properties.
+    6. Use whole words in names when possible.
+
+  ### Types
+    1. Do not export types/functions unless you need to share it across multiple components.
+    2. Do not introduce new types/values to the global namespace.
+    3. Shared types should be defined in `types.ts`.
+    4. Within a file, type definitions should come first.
+
+  ### `null` and `undefined`
+    1. Use `undefined`. Do not use null.
```

Comment:
What's the idea behind this?

Reply:
Sometimes there are functions that accept parameters as null or object and although we could send a null in some variables where we control the flow, sometimes other code might send undefined or simply not send anything thus making the parameters undefined. This makes us do a check for both, param==null || param == undefined . We can avoid the extra check for null.